src/Data/Nat/DivMod.agda

```diff
@@ -223,6 +228,32 @@ m≥n⇒m/n>0 {m@(suc m-1)} {n@(suc n-1)} m≥n = begin
   n / d + (m * n) / d ≡⟨ cong (n / d +_) (*-/-assoc m d∣n) ⟩
   n / d + m * (n / d) ∎
 
+/-step : ∀ {m n n≢0} → m ≥ n → (m / n) {n≢0} ≡ 1 + ((m ∸ n) / n) {n≢0}
+/-step {zero} {zero} {()} _
+/-step {zero} {suc n} {_} ()
+/-step {suc m} {suc n} {n≢0} m≥n = begin-equality
```

Comment:
I'd use `{m@(suc m-1)}` and `{n@(suc n-1)}` instead of `{suc m}` and `{suc n}` so you can avoid writing `suc` everywhere!

Reply:
I was looking for a way to avoid all those `suc`s actually! Didn't know there was such an argument alias construct... Nice